housekeeper/cli/delete.py

```diff
@@ -134,6 +134,28 @@ def files_cmd(context, yes, tag, bundle_name, before, notondisk, list_files, lis
             delete_file(file=file, store=store)
 
 
+@delete.command("tag")
+@click.option("-n", "--name", help="Name of the tag to delete")
+@click.option("-y", "--yes", is_flag=True, help="skip checks")
+@click.pass_context
+def tag_cmd(context, yes, name: str):
+    """Delete a tag from the database.
+    Entries in the file_tag_link table associated with the tag will be removed."""
+    store: Store = context.obj["store"]
+    tag: Tag = store.get_tag(tag_name=name)
```

Comment:
Type hint should be `Optional` if it can be None

Reply:
According to [PEP 604](https://peps.python.org/pep-0604/#proposal) Is preferred to use `Tag | None`. I will add this and resolve the conversation.